README.md

```diff
@@ -1,70 +1,37 @@
-
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
-
-### Analyzing the Bundle Size
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
-
-### Making a Progressive Web App
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
-
-### Advanced Configuration
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+A web app developed using React. It's a simple calcualtor application. It also has a small router for about pages. Other than calculator all pages are static components.
+
+Built With
+HTML5
+CSS3
+React JS
+Live Demo (if available)
+Live Demo Link
+
+Getting Started
+git clone https://github.com/bigmosi/magicians.git
+cd magicians
+npm run install
+npm run start
+Authors
+👤 **Kinyera Amos**
+
+- GitHub: [@bigmosi](https://github.com/bigmosi/)
+- Twitter: [@kinyera_amos](https://twitter.com/kinyera_amos)
+- LinkedIn: [kinyera-amos-026062218](https://linkedin.com/in/kinyera-amos-026062218)
+
+handshake Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a star if you like this project!
+
+Acknowledgments
+Microverse
+React
+memo License
+This project is MIT licensed.
+
+Code Reviewers
+Francis Ugorji @GitHub
```

Comment:
- [ ] Good job so far, however, kindly make use of the Microverse Readme [template](https://github.com/microverseinc/readme-template/blob/master/README.md) to structure your Readme. This will provide the necessary markdowns and headings that will make your Readme more readable and more professional. Please personalize it with the content you have here already.